mpmath/functions/zeta.py

```diff
@@ -447,6 +447,9 @@ def polylog_unitcircle(ctx, n, z):
     return l
 
 def polylog_general(ctx, s, z):
+    if(ctx.isinf(s) or ctx.isnan(s)):
```

Comment:
Then, this should be removed.  (BTW, please try to follow the PEP 8 for new code.)